docs/diagnostics/experimental-apis/README.md

```diff
@@ -58,3 +52,11 @@ Description: Metrics Exemplar Support
 Details: [OTEL1002](https://github.com/open-telemetry/opentelemetry-dotnet/blob/b8ea807bae1a5d9b0f3d6d23b1e1e10f5e096a25/docs/diagnostics/experimental-apis/OTEL1002.md)
 
 Released stable: `1.9.0`
+
+### OTEL1003
```

Comment:
Does this need to be moved?

Reply:
I was simply following the convention of the OTEL1002 above which was released and moved from "Active" to "Inactive". Personally I think it'd also be helpful to have a record of what happened to previous experimental features.